src/js/App/Activation/Activation.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import Cookies from 'js-cookie';
+import PropTypes from 'prop-types';
+
+const Activation = ({ user }) => {
+  const [isModalOpen, setIsModalOpen] = useState(true);
+  const history = useHistory();
+  const env = window.insights.chrome.getEnvironment();
```

Comment:
can we use the `getEnv` function directly from `utils.js` instead of accessing it from window?